tests/PySys/monitoring/monitoring_smaller_interval/run.py

```diff
@@ -0,0 +1,122 @@
+from pysys.basetest import BaseTest
+
+import time
+import json
+
+"""
+Validate tedge-dm-agent  messages that are published
+on tedge/measurements
+
+Given a configured system
+When we start the tedge-dm-agent with sudo in the background
+When we start tedge sub with sudo in the background
+When we start two publishers to publish the simulated collectd messages
+Publish the messages in 100ms interval
+Wait for couple of seconds to publish couple of batch of messages
+Then we kill tedge sub with sudo as it is running with a different user account
+Then we validate the  messages in the output of tedge sub,
+
+"""
+
+
+class MonitoringSmallInterval(BaseTest):
+    def execute(self):
+        tedge = "/usr/bin/tedge"
+        sudo = "/usr/bin/sudo"
+
+        # stop collectd to avoid mixup of  messages
+        collectd = self.startProcess(
+            command=sudo,
+            arguments=["systemctl", "stop", "collectd"],
+            stdouterr="collectd",
+        )
+
+        collectd_mapper = self.startProcess(
+            command=sudo,
+            arguments=["systemctl", "start", "tedge-dm-agent"],
+            stdouterr="collectd_mapper",
+        )
```

Comment:
The above two steps(stopping collectd and starting dm-agent) can go into the `setup` phase/method of pysys and a cleanup function needs to be registered to stop this `tedge-dm-agent` service at the end of the test so that subsequent tests are not affected.

Reply:
done